api/v1beta1/verticascrutinize_types.go

```diff
@@ -172,3 +179,19 @@ func MakeVscr() *VerticaScrutinize {
 		},
 	}
 }
+
+func (vscr *VerticaScrutinize) GetLabels() map[string]string {
```

Comment:
Why does this function (and GetAnnotations) do a copy of the map? Also, if this is needed still we can use `maps.Clone` from the standard library instead of this.

Reply:
It is use when building the pod spec to set labels(annotations) on pod metadata. I used them to avoid a shallow copy. I will use maps.Clone

Reply:
`maps` package is supported from go1.21. We are still running go1.20 in the operator container. So I am going to keep this until we upgrade to go1.21.

Reply:
Yeah, makes sense. Maybe leave a comment indicating that we can replace this when moving to 1.21. Could we call the functions `CopyLabels/CopyAnnotations`? I think we should be clear that there is a cost to this. I don't want thinking its a cheap getter function.